src/cli.ts

```diff
@@ -128,55 +128,78 @@ async function main() {
     serverRoot: flags.serverRoot,
   };
   if (flags.skip) {
-    if (typeof flags.skip === 'string') {
-      opts.linksToSkip = flags.skip.split(' ').filter(x => !!x);
-    } else if (Array.isArray(flags.skip)) {
-      opts.linksToSkip = flags.skip;
-    }
```

Comment:
FYI: removing the support for `skip` to be a list is a breaking change (when using the `linkinator.config.json` config file).

Was this intentional?

Reply:
looks like a bug :)